Riot/AppDelegate.m

```diff
@@ -48,10 +46,16 @@
 #import <MatrixEndpointWrapper/MatrixEndpointWrapper.h>
 #endif
 
-#include <MatrixSDK/MXJingleCallStack.h>
-
 #include <MatrixSDK/MXUIKitBackgroundModeHandler.h>
 
+// Calls
+#import "CallViewController.h"
+
+#import <MatrixSDK/MXCallKitAdapter.h>
+#import <MatrixSDK/MXCallKitConfiguration.h>
+#import <MatrixSDK/MXJingleCallStack.h>
```

Comment:
To be aligned with the rest of code, we can put libJingle headers under the MX_CALL_STACK_JINGLE compilation flag.